bgpd/bgp_nht.c

```diff
@@ -709,7 +709,24 @@ static void bgp_nht_ifp_table_handle(struct bgp *bgp,
 	bool nhop_other = false;
 
 	frr_each (bgp_nexthop_cache, table, bnc) {
-		if (bnc->ifindex_ipv6_ll != ifp->ifindex)
+		for (nhop = bnc->nexthop; nhop; nhop = nhop->next) {
+			if (nhop->ifindex == ifp->ifindex &&
+			    nhop->vrf_id == ifp->vrf->vrf_id)
+				nhop_found = true;
```

Comment:
This variable is added in https://github.com/FRRouting/frr/pull/13509/commits/0d9cc032541d235aa76e537668e9c31de26ba100 which is nothing really to do. Can we make it belong to the same commit as this change? In terms of backports, this gonna be a conflict.

Reply:
of course. my bad.